packages/relay/tests/lib/precheck.spec.ts

```diff
@@ -429,33 +431,50 @@ describe('Precheck', async function () {
     const contractCall = '0xcfae3217';
     const transfer = '0x';
     const invalidTx = '0x60806040523480156200001157600080fd5b';
-    it('should be able to calculate contract create', function () {
-      const intrinsicGasCost = Precheck.transactionIntrinsicGasCost(contractCreate);
+    it('should be able to calculate small contract create', function () {
+      const mirrorNodeEstimation = 60364;
```

Comment:
How did this number for the small contract create come about?

Reply:
Created the contract with Remix, tried to deploy on previewnet and got the number from the mirror-node.

Reply:
@georgi-l95 you can add a really simple comment on top of the estimations here and below to show future developers what call to make to the mirror node to get the value of `mirrorNodeEstimation` in case it ever changes
```suggestion
      // curl -X POST --data '{"jsonrpc":"2.0","id":1,"method":"eth_call","params":[{"from":"0x...","data":<greeterContractCreate>},"latest"]}'
      const mirrorNodeEstimation = 60364;
```